runtime/compiler/env/j9method.cpp

```diff
@@ -5666,7 +5666,7 @@ uint32_t
 TR_ResolvedJ9Method::classCPIndexOfMethod(uint32_t methodCPIndex)
    {
    uint32_t realCPIndex = jitGetRealCPIndex(_fe->vmThread(), romClassPtr(), methodCPIndex);
-   uint32_t classIndex = ((J9ROMMethodRef *) cp()->romConstantPool)[realCPIndex].classRefCPIndex;
```

Comment:
This change doesn't seem connected to the goal of this PR. Moreover, it's change for non-jitserver. Is it required?

Reply:
This change allows us to remove JITServer version of `classCPIndexOfMethod`. It's related to ROM classes so I put it here, but it's not required to go into this PR.